components/statuslight/stories/statuslight.stories.js

```diff
@@ -54,15 +61,71 @@ export default {
 		size: "m",
 		label: "Status",
 		variant: "info",
+		isDisabled: false,
 	},
 	parameters: {
 		packageJson: pkgJson,
 	},
 };
 
+/**
+ * Status lights should always include a label with text that clearly communicates the kind of status being shown. Color alone is not enough to communicate the status. Do not change the text color to match the dot.
+ * 
+ * When the text is too long for the horizontal space available, it wraps to form another line.
+ */
 export const Default = StatusLightGroup.bind({});
 Default.args = {};
 
+/**
+ * Status lights come in four different sizes: small, medium, large, and extra-large. The medium size is the default and most frequently used option. Use the other sizes sparingly; they should be used to create a hierarchy of importance within the page.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withBorder: false,
+	withHeading: false,
+	...args,
+}, context);
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disabledSnapshot: true },
+};
+
+/**
+ * When status lights have a semantic meaning, they use semantic colors. Use these variants for the following statuses:
+ * - Informative (active, in use, live, published)
+ * - Neutral (archived, deleted, paused, draft, not started, ended)
+ * - Positive (approved, complete, success, new, purchased, licensed)
+ * - Notice (needs approval, pending, scheduled, syncing, indexing, processing)
+ * - Negative (error, alert, rejected, failed)
+ * 
+ * Semantic status lights should never be used for color coding categories or labels, and vice versa.
+ */
+export const SemanticColors = SemanticGroup.bind({});
+SemanticColors.tags = ["!dev"];
+SemanticColors.parameters = {
+	chromatic: { disabledSnapshot: true },
+};
+SemanticColors.storyName = "Semantic colors";
+
+/**
+ * When status lights are used to color code categories and labels that are commonly found in data visualization, they use label colors. The ideal usage for these is when there are 8 or fewer categories or labels being color coded.
+ */
+export const NonSemanticColors = NonsemanticGroup.bind({});
+NonSemanticColors.tags = ["!dev"];
+NonSemanticColors.parameters = {
+	chromatic: { disabledSnapshot: true },
+};
+NonSemanticColors.storyName = "Non-semantic colors";
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+	isDisabled: true,
+};
+Disabled.tags = ["!dev"];
+Disabled.parameters = {
+	chromatic: { disabledSnapshot: true },
+};
```

Comment:
We specifically removed the disabled styles and the disabled variant from the docs page in #1495. There's also no disabled status lights in S2. The only reason I added this is because the guidelines page references a disabled status light state. Perhaps that's out of date?

I have a separate commit for the disabled story and testing for easy removal. If we decide to keep the disabled stuff, I have a draft PR here: #3153 to add the disabled styles back into our CSS.